scarb-metadata/src/command/metadata_command.rs

```diff
@@ -142,6 +150,9 @@ impl MetadataCommand {
         let mut cmd = self.scarb_command();
         let output = cmd.output()?;
         if !output.status.success() {
+            if self.inherit_stdout {
+                stdout().write_all(&output.stdout)?;
+            }
```

Comment:
Shouldn't we be able to pass stdout messages when the command is successful as well?

Reply:
If command is successful we return it's output parsed as `Metadata` so I don't see the point in printing it.

Reply:
Even if the command is successful, the metadata itself is just one of messages sent to stdout. The other may include resolver warning, or info like the one below. 
```
    Updating git repository https://github.com/keep-starknet-strange/alexandria
{
  "version": 1,
``` 
So imho it would be cool to forward them as well, for full functionality of stdout inheritance (except for the metadata one ofc).